sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.MainProcessingLoop.cs

```diff
@@ -562,10 +562,11 @@ public async Task BackgroundProcessingStartsProcessingForClaimedPartitions()
         /// </summary>
         ///
         [Test]
+        [Timeout(300_000)] // TEMP:  Using 5 minutes as an arbitrary timeout while troubleshooting a suspected test hang.
```

Comment:
This test was flagged as the most likely candidate for being the cause of the hang, according to the run data that I was able to parse.   This attribute should allow the test runner to hard-abort after 5 minutes and mark the test as a failure rather than hanging.   

The suite gets angry with the timeout applied too wide, but seems stable with it focused on a single test or two.

Reply:
What is the format this timeout value is using?